New Get-ITGlueLogs & Added support for the Australian datacenter

Celerium opened the pull request:
Apologies as I am new to Git\GitHub, please let me know if I am doing this wrong and ill do my best to fix it.

1. Added support to get [ITGlue logs](https://api.itglue.com/developer/#logs-index). 
- #137 
- The sort parameter allows for 'created_at', 'updated_at', '-created_at', '-updated_at'. Though in my testing I am unsure what the 'updated_at' &  '-updated_at' are pulling from as there are no values in the returned data nor are they options in any of ITGlue's logs graphically. The data does sort correctly though when the options are inputted.

2. Added support for the Australian datacenter

- I am unable to fully test as my ITGlue instance is US-based

3. Fixed various minor spelling mistakes & trailing spaces. 

4. Updated ModuleVersion to 2.2.0 & adjusted any related items for function imports\tests.

davidhaymond commented:
Great PR. Thanks for the trailing whitespace cleanup. My only comment is that the [documentation](https://api.itglue.com/developer/#logs-index) lists `created_at` only, so `updated_at` and `-updated_at` should be removed from the list of allowed sort values.

CalebAlbers reviewed (commented):
Thanks for the contribution, @Celerium! Both the logs endpoint and support for AU are really impactful features

CalebAlbers commented on ITGlueAPI/ITGlueAPI.psd1:
```diff
@@ -18,7 +18,7 @@
     # -- PATCH version when you make backwards-compatible bug fixes.
 
-    ModuleVersion = '2.1.0'
+    ModuleVersion = '2.2.0'
```
@davidhaymond made me aware via #148 that my own instructions from a few years back asked for a ModuleVersion bump, so that's entirely on me 🙃 

I went ahead and committed the change to revert the bump and will merge this PR once the other notes about `updated_at` and -`updated_at` are addressed.

Thanks!

Celerium commented:
Weird I could have sworn those were there at some point. Either way, I updated the parameters, tested them, and added comment-based help. There is another parameter that can be added in the future.

[ITGlue Logs](https://api.itglue.com/developer/#logs-index)
- filter[created_at]
- It would have to be a DateTime input from the looks of it that could also accept a wildcard of '*'.

CalebAlbers commented:
Thanks again for this! Loving the comment-based help, too!
